docs/filter_format.rst

```diff
@@ -74,121 +74,54 @@ Plain
 _____
 Filter is based representation of the mechanism of
 replacing modules, functions or decorating functions.
```

Comment:
I am worried that this "filter" concept is a bit misleading and mostly irrelevant to the users.

Reply:
I understand, just as I've said, maybe this documentation would be better to do from scratch. I've just wanted to remove at least parts what are complety changed